scipy/stats/_qmc.py

```diff
@@ -138,64 +147,66 @@ def discrepancy(sample, iterative=False, method='CD'):
     >>> disc_init = qmc.discrepancy(space[:-1], iterative=True)
     >>> disc_init
     0.04769081147119336
+    >>> _update_discrepancy(space[-1], space[:-1], disc_init) # doctest: +SKIP
```

Comment:
Private functions (indicated by `_`) should not be shown in public documentation.